src/Form/Form.js

```diff
@@ -510,7 +510,8 @@ class Form {
             if (!canBeInteractedWith(input)) return
 
             // Checks for pointerdown event. Needed for positioning when fields are within an iframe
-            if (e.type === 'pointerdown') {
+            // TODO: Confirm this is an ok change -- by not calling `getMainClickCoords` we're no longer checking if the event is trusted
```

Comment:
Added this `todo` here as the comment didn't match the implementation. I've updated it to check for fields within an iframe, but I want to make sure we're not losing other checks by doing this.